src/web/mjs/connectors/BilibiliManhua.mjs

```diff
@@ -13,7 +13,11 @@ export default class BilibiliManhua extends Connector {
         this.auth = {
             accessToken : '',
             refreshToken : '',
+            area : 1
```

Comment:
Maybe use `area` variable as string `area: this.areacode[1];`

Reply:
> Maybe use `area` variable as string `area: this.areacode[1];`

Good idea, but its saved as a number in cookies so i'd rather keep the same data type. It may make further fixes easier.